packages/account/src/Components/poa/common-mistakes/__tests__/common-mistake-example.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
file name should be `common-mistake-examples` to follow the rest of the file names.